time t
float32 x
float32 y
float32 z
float32 a
float32 b
float32 c
float32 diff_x
float32 diff_y
float32 diff_z
float32 diff_a
float32 diff_b
float32 diff_c
float32 ddiff_x
float32 ddiff_y
float32 ddiff_z
float32 ddiff_a
float32 ddiff_b
float32 ddiff_c
